packages/amplify_datastore/ios/Classes/types/model/FlutterModelField.swift

```diff
@@ -68,17 +104,39 @@ public struct FlutterModelField {
         //        }
         preconditionFailure("Could not create a ModelFieldType from \(String(describing: type)) MetaType")
     }
-    
-public func convertToNativeModelField() -> ModelField {
+
+    private func convertFlutterAuthRules(flutterAuthRules : [FlutterAuthRule]?) -> [AuthRule]?{
+
+        if(flutterAuthRules != nil){
```

Comment:
`guard else` and return early.

Reply:
This code has been updated to : `        if let flutterAuthRules = flutterAuthRules{
`